local-run/application.yaml.template

```diff
@@ -0,0 +1,36 @@
+#
+# Indicate the host and port for the Symphony POD the Apps are associated with.
+#
+pod:
+  host: <<POD_HOST>>
+  port: <<POD_PORT>>
+
+#
+# Indicate the host and port for the POD's Agent.
+#
+agent:
+  host: <<AGENT_HOST>>
+  port: <<AGENT_PORT>>
+
+#
+# Indicate the host and ports for the POD's Key Manager.
+#
+key_manager:
+  host: <<KM_HOST>>
+  port: <<KM_PORT>>
+
+key_manager_auth:
+  host: <<KM_AUTH_HOST>>
+  port: <<KM_AUTH_PORT>>
+
+pod_session_manager:
+  host: <<SM_HOST>>
+  port: <<SM_PORT>>
+
```

Comment:
@maoo We can simplify this template just changing the env.sh script.

This template should contain only the application info and profiles. The API endpoints you could provide on the env.sh, something like that:

```
spring:
  profiles:
    active: <<APP_ID>>

applications:
  <<APP_ID>>:
    state: PROVISIONED
    keystore:
      file: <<CERT_FILE>>
      password: <<CERT_PWD>>
    type: pkcs12
```